docs/users.rst

```diff
@@ -113,6 +131,59 @@ not
   list of platform names (or a single platform name as a string);
   this restricts the platforms to build for using set difference
 
+compose
+~~~~~~~
+
+This section is used for requesting yum repositories at build time. When this
+section is defined, a compose will be requested by using ODCS.
+
+packages
+  list of package names to be included in ODCS compose. Package in this case
+  refers to the "name" portion of the NVR (name-version-release) of an RPM, not
+  the Koji package name. Packages will be selected based on the Koji build tag
+  of the Koji build target used. The following command is useful in determining
+  which packages are available in a given Koji build tag:
+  ``koji list-tagged --inherit --latest TAG``
+
+pulp_repos
+  boolean to control whether or not an ODCS compose of type "pulp" should be
+  requested. If set to true, ``content_sets.yml`` must also be provided. A
+  compose will be requested for each architecture in ``content_sets.yml``.
+
+modules
+  list of modules for requesting ODCS compose of type "module".
+
+signing_intent
+  used for verifying packages in yum repositories are signed with expected
+  signing keys. The possible values for signing intent are defined in OSBS
+  environment. See :ref:`config.yaml-odcs` section for environment configuration
+  details, and full explanation of :ref:`signing-intent`.
+
+**If "compose" section is defined, non-empty list of "modules" or "packages" must
+be provided, or "pulp_repos" set to true. Otherwise, build will fail.**
+
+**The "packages" and "modules" keys are mutually exclusive. If both are
+provided, "modules" will be ignored. "pulp_repos" can be used by itself, or with
+either "packages" or "modules".**
+
+
+Content Sets
+------------
+
+This files is used to define the content sets relevant to the container image.
```

Comment:
Oh, here it is! :)

How about rephrasing this to say: "`content_sets.yaml is used to define...", so that when a search for `content_sets.yml` is made, this shows up as a result.

Reply:
Fixed